javascript/stricjs/package.json

```
@@ -1,5 +1,5 @@
 {
   "dependencies": {
-    "@stricjs/router": "4.0.9"
+    "@stricjs/router": "4.0.10"
```

I suggest using the last patch available

```suggestion
    "@stricjs/router": "~4.0.10"
```

---

Thanks that looks good